bin/varnishstat/varnishstat.c

```diff
@@ -136,9 +134,12 @@ do_json(struct vsm *vsm, struct vsc *vsc)
 	now = time(NULL);
 
 	(void)strftime(time_stamp, 20, "%Y-%m-%dT%H:%M:%S", localtime(&now));
-	printf("{\n  \"timestamp\": \"%s\",\n", time_stamp);
+	printf("{\n"
+	    "  \"version\": 1,\n"
```

Comment:
I'd be tempted to start right away with version 2.

Reply:
if you want, but as said, most parser will consider that a missing `version` field means `version=0`, so I found that relatively pleasing.

But we are in bikeshedding territory, so I don't care tooo much

Reply:
Then let's make it obvious to actual end users that this is a new version and bump it to 2.

Reply:
My vote goes to zero-based: The current format is version NIL ≍ 0, so the next version is 1.

Reply:
bugwash decision: @gquintard you get to break the tie here. Please decide on the initial version number, then the PR is ready for merge in april.

Reply:
My opinion has not changed, and is reflected by the current code. I don't care enough veto a version 2 though, so @Dridi if you want to update the PR, please go ahead

Reply:
d-oh @gquintard I appreciate your kindness, but we tried to break the tie... Now If you do not make a decision, aren't we back where we started?

Reply:
@nigoroll : my position is that we should start at `1`, as the code currently says, meaning I tip the scales in your favor. Of did I misread the situation?

Reply:
no, you did not.